src/main/java/com/cegeka/academy/service/util/SortUtil.java

```diff
@@ -0,0 +1,27 @@
+package com.cegeka.academy.service.util;
+
+import com.cegeka.academy.service.dto.UserDTO;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SortUtil {
+
+    public static List<UserDTO> sortUsersByDate(List<UserDTO> users) {
+
+        return users.stream().sorted((o1, o2) -> {
+            if (o1.getCreatedDate() == null || o2.getCreatedDate() == null)
```

Comment:
Daca un user nu are `createdData` atunci avem un bug, dar e bine sa ne gardam de aceasta situatie. Ar fi bine sa logam cazul acesta.
Un user cu data de creare nula nu poate fi "egal" cu un alt user decat daca are si el data de creare nula, deci primul `if` ar trebui ajustat.
Compara datele folosind metode existent in java. Vezi documentatie de [aici](https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html)

Reply:
Exista 2 modatitati de a compara obiecte in java si anume orin intermediul a 2 interfete `Comparable` si `Comparator`.

Reply:
Ma gandesc ca daca folosesc una din interfete trebuie sa modific modelul. Nu este acelasi lucru daca am pus comparatorul intr-o clasa anonima?